tslearn/svm.py

```diff
@@ -71,6 +71,15 @@ def _preprocess_sklearn(self, X, y=None, fit_time=False):
         else:
             return sklearn_X, y
 
+    def _check_test_data(self, X):
+        n_samples, n_seqlen, n_features = X.shape
```

Comment:
You can take the last dimension only if you use only this one, as you did it in the next line.